ui/src/UI/Resources/Localization/Translations/en-us.ftl

```diff
@@ -72,6 +72,11 @@ tray-disconnected = Disconnected
 tray-unstable = Unstable
 tray-no-signal = {hero-subtext-no-signal}
 
+## Tray menu
+tray-menu-exit = E_xit
```

Comment:
why E_xit? Previously it was _Exit. It might affect the UI test cuz UI test is using _Exit as Name property of the Exit menu item.

Reply:
On Windows, in English, Exit **always** has "X" highlighted as the keyboard shortcut (guess the symbol is synonymous with "close"). I propose we leave the underlined "X".